docs/Development Concepts/6.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Security Features
+---
+
+Each protocol layer of Waku v2 provides a distinct service and is associated with a separate set of security features and concerns.
```

Comment:
Add that the security models are detailed in the RFC of the protocols.
That we strive to provide well documented and open source RFCs so that Waku users know what security guarantees are and aren't provided by each protocol.